pkg/lightning/backend/remote/chunk.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2025 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package remote
+
+import (
+	"fmt"
+	"hash/crc32"
+	"os"
+	"path/filepath"
+
+	"github.com/pingcap/errors"
+)
+
+type chunkMeta struct {
+	size     int
+	checksum uint32
+	// if usingMem is true, chunkData will be used.
+	chunkData []byte
```

Comment:
fieldalignment linter should recommend to move fields containing pointer at the beginning of struct 🤔  not sure if this rule still works now